common/thrift_router.h

```diff
@@ -377,7 +382,8 @@ class ThriftRouter {
         const Role role,
         const unsigned rotation_counter,
         const std::string& segment,
-        const int shrink_target) {
+        const int shrink_target,
+        const std::string& specific_az) {
       std::vector<const Host*> v;
       if (role == Role::ANY && !FLAGS_always_prefer_local_host) {
```

Comment:
Does this mean `FLAGS_always_prefer_local_host` should be True to be able to get clients by certain AZ/PG?

Reply:
yes that was the idea. But I can make it generic and have this as the first condition. Would you like me to do that?